aten/src/ATen/native/native_functions.yaml

```diff
@@ -4971,7 +4971,7 @@
 - func: quantize_per_channel(Tensor self, Tensor scales, Tensor zero_points, int axis, ScalarType dtype) -> Tensor
   variants: function
   dispatch:
-    CPU: quantize_per_channel_cpu
+    CPU, CUDA: quantize_per_channel_cpu
```

Comment:
looks wrong to me

Reply:
I fixed this in a later diff in the stack. This pr was getting long and to make the necessary change, I have to change all calls to quantize_per_channel_cpu